theano/gof/op.py

```diff
@@ -1381,7 +1381,7 @@ def get_op_params(self):
         The names must be strings that are not a C keyword and the
         values must be strings of literal C representations.
 
-        If op uses a :class:`theano.gof.wrapper.Wrapper` as ``params_type``,
+        If op uses a :class:`theano.gof.Wrapper` as ``params_type``,
```

Comment:
The class name for documentation purposes is still theano.gof.wrapper.Wrapper so don't change that here.